### PR TITLE
Creates standalone pages for OpenSearch Dashboards RPM/YUM Installation

### LINKED ISSUE #1112: [DOC] Move dashboards install (rpm/yum) to new page in dashboards section
**What do you want to do?**
 
Currently there isn't a page for installing OpenSearch Dashboards via RPM/YUM. Rather, those instructions (only a few lines) are dwelling in the OpenSearch (core) doc, and they're presented as if they're required for OpenSearch to work. The steps are barebones, as with the rest of the Dashboards installation methods, but they should have a dedicated page under Dashboards.

**Tell us about your request.**

This affects all versions.
 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### PR DESCRIPTION
**Pull request by JeffH-AWS**
Signed-off-by: JeffH-AWS <jeffhuss@amazon.com>

### Description
Previously, steps for installing OpenSearch Dashboards by way of RPM/YUM were not included under the OpenSearch Dashboards section. Rather, they were interlaced with OpenSearch (core) installation instructions. This is confusing for users. This PR extracts and reformats the existing installation steps for OpenSearch Dashboards, without conflating plugins with the core product.

These installation steps are MORE COMPLETE and MORE ACCURATE than what was originally included. This PR does not represent a final form for these steps. However, they need to live somewhere and there isn't room for them in the refactored RPM/YUM installation doc for OpenSearch (core).

### Issues Resolved
Fixes #1112 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/install/rpm.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+title: RPM
+parent: Install OpenSearch Dashboards
+nav_order: 31
+---
+
+# Run OpenSearch Dashboards using RPM Package Manager (RPM)
+
+OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+
+1. Download the RPM package directly from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}. The RPM package can be download for both `x64` and `arm64` architectures.
```
Comment (vagimeli) [inline _dashboards/install/rpm.md:12]: Change "download" to "downloaded"

File: _dashboards/install/rpm.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+1. Import the public GPG key. This key verifies that the your OpenSearch instance is signed.
```
Comment (vagimeli) [inline _dashboards/install/rpm.md:13]: Delete "the" before "your"

File: _dashboards/install/rpm.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+   sudo curl -SL https://artifacts.opensearch.org/releases/bundle/opensearch/2.x/opensearch-2.x.repo -o /etc/yum.repos.d/opensearch-2.x.repo
+   ```
+   ```bash
+   sudo curl -SL https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/2.x/opensearch-dashboards-2.x.repo -o /etc/yum.repos.d/opensearch-dashboards-2.x.repo
+   ```
+1. Verify that the repository was created successfully.
+    ```bash
+    sudo yum repolist
+    ```
+1. Clean your YUM cache, to ensure a smooth installation:
```
Comment (vagimeli) [inline _dashboards/install/rpm.md:52]: Delete comma after "cache"

File: _dashboards/install/rpm.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+   ```bash
+   sudo yum clean all
+   ```
+1. With the repository file downloaded, list all available versions of OpenSearch-Dashboards:
```
Comment (vagimeli) [inline _dashboards/install/rpm.md:56]: Does the hyphen belong between OpenSearch and Dashboards?

**Review by vagimeli — APPROVED** [review]
A few minor edits

File: _dashboards/install/rpm.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+layout: default
+title: RPM
+parent: Install OpenSearch Dashboards
+nav_order: 31
+---
+
+{% comment %}
+The following liquid syntax declares a variable, major_version_mask, which is transformed into "N.x" where "N" is the major version number. This is required for proper versioning references to the Yum repo.
+{% endcomment %}
+{% assign version_parts = site.opensearch_major_minor_version | split: "." %}
```
Comment (Naarcha-AWS) [inline _dashboards/install/rpm.md:11]: This is cool!

**Review by Naarcha-AWS — APPROVED** [review]

**Comment by JeffH-AWS** [comment]
@Naarcha-AWS I've turned on the bat signal and requested help with the 1.3 backport (see Slack).

**Pull request merged**